components/fieldgroups/shared/forecasted-weather.schema.json

```diff
@@ -324,6 +240,18 @@
           "description": "Distance in miles."
         }
       }
+    },
+    "uvIndex": {
+      "properties": {
+        "xdm:_name": {
```

Comment:
what is the significance of "_" in "xdm:_name" ? Can this be simple name without "_" ?

Reply:
@prabhum2 - we need this in the schema to align with some data we're producing and unfortunately is much more expensive for us to adjust the data we're producing